src/main/fc/rc_adjustments.c

```diff
@@ -657,6 +663,13 @@ static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, ui
                 pidAudioSetMode(newMode);
             }
         }
+#endif
+        break;
+#ifdef USE_OSD_PROFILES
+    case ADJUSTMENT_OSD_PROFILE:
+        if (getCurrentOsdProfileIndex() != (position + 1)) {
```

Comment:
The SITL target breaks on this. I think the right way to address this is to add 

```
#undef USE_OSD_PROFILES
```

to the

```
#if !defined(USE_OSD)
```

stanza in `src/main/target/common_post.h`.

Reply:
Thanks.  Now fixed.